vitest-dev/vscode
# Support root config

## Done

- Add `rootPath` to support projects that are not at the root of the workspace (fixes: #151, fixes: #49).
- Support the Vitest `root` config for tests that are not at the root of a project.

## QA

- Run this extension with the Nested Root Sample project.
- It should find and run the `add.test.ts` tests.
